app/controllers/iiif/auth/v2/probe_service_controller.rb

```diff
@@ -70,10 +85,6 @@ def unauthorized_heading(file)
             I18n.t('probe_service.stanford')
           elsif file.stanford_restricted? && file.embargoed?
             I18n.t('probe_service.stanford_and_embargoed', date: file.embargo_release_date.to_date)
-          elsif file.embargoed?
```

Comment:
Why drop embargoed?  Aren't there embargos where the file is stanford-only and you can login to see it.  When the embargo is lifted it will be world readable.

Reply:
The two lines before check for stanford_restricted and embargoed. We moved the embargoed into forbidden_fields because if it gets to embargoed then the file is embargoed and not stanford restricted which means no one can look at it.

Reply:
Ah. I see now.